app/views/backend/parcels/_form.html.haml

```diff
@@ -24,6 +24,7 @@
   = f.input :given_at, required: false
   - if f.object.outgoing?
     = f.input :remain_owner
+  = f.association :creator, input_html: { value: current_user.id }, as: :hidden
```

Comment:
Risky. If we need this, it needs to be moved up into the controller.
Also: don't timestamps already handle this?